e2e/generate-ccds.js

```diff
@@ -0,0 +1,32 @@
+const fs = require('fs');
+const util = require('util');
+const bbg = require('..');
+
+(async () => {
+  getFiles().then((filePaths) => generateCCDs(filePaths));
+})();
```

Comment:
I'm not sure if we have a style for this, but I've seen our code base place these "main functions" at the bottom of the file. Might want to talk it over with someone.